### DISCUSSION TITLE
Updates for Strapi v5 and Medusa v2?

### DISCUSSION THREAD

**Discussion by Deroswent** [SGFGOV/medusa-strapi-repo#87] [General]
Is there any plans to further develop this plugin and update it for use in new versions Strapi v5 and Medusa v2? Or is it a dead project?

**Comment by SGFGOV** [comment]
I think a wider discussion is necessary.. lots of things changed.. what are
the features and why a cms?.. this is the question I am trying to
understand first

On Thu, 24 Oct, 2024, 5:58 pm N1on, ***@***.***> wrote:

> Is there any plans to further develop this plugin and update it for use in
> new versions Strapi v5 and Medusa v2? Or is it a dead project?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SGFGOV/medusa-strapi-repo/discussions/87>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHFLRLWQW4EOGCKOT5TZ5DRYHAVCNFSM6AAAAABQQ7GGRKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGM3DENJSGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.

**Reply by Deroswent** [reply]
Hello. Thank you for your reply. 

I am a developer with several projects on Medusa, and I can clearly answer why a CMS is needed: Medusa developers have repeatedly said that they do not plan to add l18n for products, rich product description editor and different images for product variants and other CMS features.  They explicitly say that Medusa was not created as a CMS and will not support these features, for this developers recommend using some CMS, and Medusa itself will act as a kind of “very advanced database, supplying data to the CMS”. 

In my opinion, Strapi is the best CMS in its class, which is why an integration plugin for new versions of Medusa and Strapi is so desirable.
